hmy/backend.go

```diff
@@ -55,6 +57,7 @@ func New(nodeAPI NodeAPI, txPool *core.TxPool, eventMux *event.TypeMux) (*Harmon
 		chainDb:        chainDb,
 		bloomIndexer:   NewBloomIndexer(chainDb, params.BloomBitsBlocks, params.BloomConfirms),
 		nodeAPI:        nodeAPI,
+		networkID:      1, // TODO(ricl): this should be from config
```

Comment:
There is a networkVersion below and here networkID.

Those 2 terms make me very confused. Can you write comments as your clarification?

Reply:
agreed it's confusing. For now it's for sticking to eth's standard. I updated the comments.